src/snowflake/snowpark/dataframe_reader.py

```diff
@@ -1144,7 +1153,7 @@ def _read_semi_structured_file(self, path: str, format: str) -> DataFrame:
                 handler_name,
                 output_schema=output_schema,
                 input_types=input_types,
-                packages=["snowflake-snowpark-python"],
+                packages=["snowflake-snowpark-python", "lxml"],
```

Comment:
do we need a cap here? like 4 < lxml < 6?
or you have tested that all versions are compatibile?

Reply:
good point, done